db/index.js

```diff
@@ -56,8 +64,29 @@ const getCharitiesById = function(id, callback) {
   });
 };
 
+
+// db.connectAsync()
+//   .then(() => {
+//     console.log(`Connected to MySQL Charities!`)
+//   })
+//   .then(() => {
+//     db.queryAsync(`CREATE DATABASE IF NOT EXISTS ${database}`)
+//   })
+//   .then(() => {
+//     db.queryAsync(`USE ${database}`)
+//   })
+//   .then(() => {
+//     conf(db)
+//   })
+  // .then(() => {
+  //   db.queryAsync(`INSERT INTO charity (id, name, description) VALUES ('1','Parker-Erdman','Sunt quod autem et et et distinctio. Aut est eum omnis mollitia explicabo quia nulla voluptatibus. Amet consequatur saepe unde iure. Corporis nam doloribus nihil quas ut consequatur. Aliquam perferendis in debitis reprehenderit dolorum.'),
```

Comment:
It is preferred to have no more than 80 characters per line for ease of code readability.
I would probably put line breaks after the commas in VALUES, it shouldn't affect how the code works, but will make it easier to parse.